content/docs/studio/user-guide/billing-and-payment.md

```diff
@@ -0,0 +1,136 @@
+# Billing and Payment
+
+Studio is free for individuals and small teams (up to 2 members). The Free plan
+is designed for teams just starting out with collaboration: you can invite one
+other team member and import as many repositories as you want. For more advanced
+collaboration functionality, you can sign up for the Team or Enterprise plans.
+These are paid plans, and in this section you will learn how to manage billing
+and payment for these plans.
+
+- [Difference between Free, Team and Enterprise plans](#difference-between-free-team-and-enterprise-plans)
+- [How to sign up for the Team or Enterprise plan](#how-to-sign-up-for-the-team-or-enterprise-plan)
+- [How total price is calculated](#how-total-price-is-calculated)
+- [How increasing or decreasing the number of seats affects total cost](#how-increasing-or-decreasing-the-number-of-seats-affects-total-cost)
+- [Downgrading a plan](#downgrading-a-plan)
+- [Billing period, auto-renewal of team plan, and grace period for non-payment](#billing-period-auto-renewal-of-team-plan-and-grace-period-for-non-payment)
+- [Payment processor and accepted payment methods](#payment-processor-and-accepted-payment-methods)
+- [Accessing your payment history and invoices](#accessing-your-payment-history-and-invoices)
+- [Frequently asked questions](#frequently-asked-questions)
+
+## Difference between Free, Team and Enterprise plans
+
+A detailed comparison of the different plans can be found in the Iterative
+Studio [pricing page](https://studio.iterative.ai/pricing).
+
+## How to sign up for the Team or Enterprise plan
+
+When you sign up, you're on the Free plan by default. Plans are managed at the
+team level, which means that you can
+[change the plan](/doc/studio/change-team-plan-or-size#change-team-plan) in a
+team's settings page. Each team is on its own plan and needs to be managed
+separately.
+
+## How total price is calculated
+
+For Enterprise plans, all invoice amounts are calculated based on the agreement
```

Comment:
This can be mentioned below the Team plans or not mentioned at all.

Reply:
Changed.